bundles/binding/org.openhab.binding.insteonplm/src/main/java/org/openhab/binding/insteonplm/internal/device/MessageHandler.java

```diff
@@ -971,16 +971,20 @@ public int transform(int raw) {
             return (raw);
         }
 
-        private int extractValue(Msg msg) throws FieldException {
+        private int extractValue(Msg msg, int group) throws FieldException {
             String lowByte = getStringParameter("low_byte", "");
-            if (lowByte == "") {
+            if (!lowByte.equals("")) {
```

Comment:
This code change has inverted the test and now (improperly) fails consistently. Insteon Smoke Bridge not working and Insteon Thermostat no longer working i.e. impossible to retrieve status and config values from thermostat during poll(?)

> `2017-03-24 00:24:36.073 [ERROR] [onplm.internal.device.MessageHandler] - NumberMsgHandler handler misconfigured, missing low_byte!